docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
+3. Measure of prevalence: 20.7% of women attending antenatal care at rural clinics Siaya county, western Kenya were HIV positive in 2015
+
+Incidence
+^^^^^^^^^
+Incidence focuses on new cases. There are two measures of incidence: risk and rate. 
+
+*Incidence risk*, also called *incidence proportion*, *attack rate*, or *cumulative incidence* is the probability of occurence of disease among a disease free, at risk, population during a specified time period. It is the number of new cases of disease during a defined period of time divided by the population at the start of the time period. It is not interpretable without specification of the time period to which it applies. The counterfactual of the incidence proportion is the survival proportion which is 1-incidence proportion.                                                 
+
+*Incidence rate* has the same numerator as incidence risk, that is the appearance of new cases. In contrast to risks, which relate the number of new cases to the size of the population at risk in the beginning of the period studied, rates relate the number of new cases to the person-time (Y) at risk, a measure that takes into account changes in the size of the population at risk during the follow-up period. The rate takes into account the fact that some people who start at risk do not remain at risk during the whole period, because they develop the disease, or die, or leave the population by migrating, refusing to continue to participate in the study etc. Others may join the population at risk after the beginning of the period, through birth, migration into the area, recruitment into the study, etc. The denominator in a rate (Y) is thus the sum of the time each person in the study population remained at risk during the study period. This is called the person-time experience at risk, and is expressed in units of person-time: person-years at risk, person-days at risk, baby-weeks at risk etc.
+
+For rare diseases, risk and rates are numerically similar. 
+
+Prevalence
+^^^^^^^^^^
+Prevalence focuses on existing states. Prevalence of a state (such as the 'with condition state') at a point in time may be defined as the proportion of a population in that state at that time; thus prevalence is the proportion of persons in a defined population that have the outcome under study in a defined period of time. 
+
+*Point prevalence* is the number of current cases (new and pre-existing) at a defined instant in time. The denominator is the population at the same defined instant in time. Eg. the percentage of people with schistosomiasis parasites in the blood in a village in Kenya in a survey in December 2019; the proportion of people who have diabetes in China today. 
```

Comment:
I think it might be good to clarify that we count this as point prevalence, i assume, because it comes from study data? or something similar? because otherwise a month sounds very much like a period.

it might be good to address that that boundary (point vs period) is somewhat ill-defined, and the definition used will probably (is this true?) be a result of the method of surveillance.

thoughts?

Reply:
I edited the example so that it so that it's a _day_ in the month of December. 

ya...i dont think when it is a long enough time span that it is considered a period vs. short enough for it to be considered a point. I'm not reading a lot of guidance on this and examples of point prevalences can span a month....